_pages/faqs.md

```diff
@@ -13,8 +13,8 @@ redirect_from:
 ---
 
 <div id="faqs-index">
-  <section class="row row-sm">
-    <div class="col-md-6">
+  <div class="syn-row syn-mb-32">
```

Comment:
Why the move away from `<section>`?

Reply:
I just feel better using `div`s for things, since they don't come with any additional styling